src/content/events/2025-06-23_kaiserslautern_arc-hackathon.md

```diff
@@ -33,5 +32,29 @@ The symposium is an on-site, in presence event held at the [Villa-Denis](https:/
 The main goal is to integrate ARC-related work of different groups across consortia. Between discussions, coding and other hands-on work, there will be various talks from metadata experts surrounding the ARC ecosystem and beyond.
 
 
+### Registration
 
+The participation fee including 2 stays and catering will be **approximately 350€** (exact price will be updated soon).
+
+**Registration is done via mail**. Single rooms will be automatically reserved for each participant. 5 weeks prior to the event, every enrolled participant will receive an invoice from the venue. The registration will be finalized through payment of this fee. 
+
+Let us know about your culinary needs.
+
+For registration, write to:
+
+Janine Mertel<br>
+<a href="javascript:location='mailto:jmertel@rptu.de">jmertel@rptu.de</a>
```

Comment:
I've opened a PR to hide the email. 
But I would recommend to handle registration via https://terminplaner6.dfn.de/de. 
There you could again reference to this site for more info, and have people "book" the event with their email.

Reply:
I'd stick to the E-Mail approach, as people need to send us their address and such anyways for receiving a correct invoice.